mythril/laser/ethereum/graph.py

```diff
@@ -0,0 +1,31 @@
+from typing import List, Dict
```

Comment:
Is there a reason why this should be in the laser.ethereum module vs laser.ethereum.strategy?

Reply:
One reason was this isn't a strategy, but this collides with the control flow graph so it might be better to move it to strategy

Reply:
moved it to strategy